app/src/androidTest/java/org/dhis2/usescases/searchte/SearchTETest.kt

```diff
@@ -307,25 +310,21 @@ class SearchTETest : BaseTest() {
         }
     }
 
-    @Ignore("To review why the sleep is needed")
     @Test
     fun shouldSuccessfullyShowMapAndTeiCard() {
         val firstName = "Lynn"
 
         prepareTBIntentAndLaunchActivity(rule)
 
         searchTeiRobot {
-            clickOnOptionMenu()
             clickOnShowMap()
             try {
-                mapIdlingResource = MapIdlingResource(rule)
```

Comment:
We should also remove the MapIdlingResource if it not going to be used.

Reply:
Thanks I'm removing it